lib/facility_factory.rb

```diff
@@ -0,0 +1,18 @@
+class FacilityFactory
+  def initialize
```

Comment:
I don't think that you even need this `initialize` method. As it is, when a `FacilityFactory` is created it has no attributes. So if you were to remove this everything should still pass.